src/docs/predicted_molecular_consequence.md

```diff
@@ -5,12 +5,88 @@ A set of information enabling interpretation of likely variant impact on genomic
 | Field             | Type            | Description
 |-------------------|-----------------|---------------------
 | allele_name           | string          | Data-derived allele name
-| feature_stable_id | string          | Feature (eg transcript) id
+| stable_id | string          | Feature stable id (transcripts for now)
 | feature_type      | OntologyTermMetadata        | SO term for feature type (eg. transcript, regulatory feature)
 | consequences      | array of OntologyTermMetadata           | SO terms for predicted impact ( eg missense variants)
-| variant_representation            | array of VariantRepresentation or [ ]  | Nomenclature values and source
+| gene_stable_id | string| Gene stable id
+| gene_symbol | string | Gene symbol
+| protein_stable_id  | string or null | Protein stable id
+| variant_representation | array of VariantRepresentation or [ ]  | Nomenclature values and source
 | prediction_results | array of PredictionResult or [ ] | Scores from programs like SIFT which calculate transcript-specific deleteriousness scores
-| relative_locations | array of VariantRelativeLocation  | 
+| transcript_biotype | string | Feature biotype
+| cdna_location | VariantRelativeLocation  | Relative location on cDNA
+| cds_location | VariantRelativeLocation or null | Relative location on CDS
+| protein_location | VariantRelativeLocation or null | Relative location on protein
+
+```
+## Example for rs699 
+
+"allele_name": "G",
+"transcript_stable_id": "ENST00000680783",
+"feature_type" : 
+{
+"accession_id": "SO:0000673",
+"value": "transcript",
+"url": "www.sequenceontology.org/browser/current_release/term/SO:0000673",
+  "source": {
+    "id": "...",
+    "name": "Sequence Ontology",
+    "url": "www.sequenceontology.org",
+    "description": "The Sequence Ontology..."
+  }
+},
+
+"consequences":
+[{
+  "accession_id": "SO:0001583",
+  "value": "missense_variant",
+  "url": "www.sequenceontology.org/browser/current_release/term/SO:0001583",
+  "source": {
+    "id": "...",
+    "name": "Sequence Ontology",
+    "url": "www.sequenceontology.org",
+    "description": "The Sequence Ontology..."
+  }
+}],
+"gene_stable_id" : "ENSG00000135744",
+"gene_symbol": "AGT",
+"protein_stable_id": "ENSP00000506329",
+"variant_representation": 
+{
+	"representation": "ENST00000680783.1:c.776T>C",
+	"naming convention": "HGVS",
+	"qualifier": "coding"
+},
+"prediction_results": [] ,
+"transcript_biotype": "protein_coding",
+"cdna_location": 
+{
+	"relation": "overlaps" ,
+	"start": 1287,
+	"end": 1287, 
+	"length": 1, 
+	"percentage_overlap": 100,
+	"sequence_change": "T/C" 
+},
+"cds_location": 
+{
+	"relation": "overlaps" ,
+	"start": 776, 
+	"end": 776, 
+	"length": 1, 
+	"percentage_overlap": 100,
+	"sequence_change": "ATG/ACG" 
```

Comment:
If you pass the value like this, then the client would need to parse this string. Far better is to parse the string on your side and to feed us with meaningful parts of it. Then you would also be able to explicitly signal that you do not have the change sequence, instead of marking it with `X`.

Perhaps:

```
"sequence_change": {
  "from": "ATG",
  "to": "ACG"
}
```

which would open up the question of how best to represent the lack of sequence or the unknown sequence.

Reply:
Thanks @azangru. A follow-up question for [long deletion example](https://xd.adobe.com/view/573e7994-6561-412f-84f0-b34c0d38761e-026d/screen/10ccfa72-1b43-45b9-ba89-87beca014c43), would you be able to parse the required information with `from` and `to`.

Reply:
> A follow-up question for [long deletion example](https://xd.adobe.com/view/573e7994-6561-412f-84f0-b34c0d38761e-026d/screen/10ccfa72-1b43-45b9-ba89-87beca014c43), would client be able to parse the required information with from and to?

Well, the question is what will be reasonable to put in the "to" field for deletions. Null? What is a good way of distinguishing between nothing in a deletion and nothing in the cases where the backend does not know, and which, as you showed, VEP expresses as "X"?

Reply:
Hi @azangru , we intend to put “-” for deletion in the "to" field and “X” for unknown/frameshift sequences
In order to avoid introducing a new type, we are proposing the following fields:
```
ref_sequence: "ATG",
alt_sequence: "ACG"
```